src/scss/components/_index.scss

```diff
@@ -3,14 +3,33 @@
 @import "badge/index";
 @import "box/index";
 @import "breadcrumb/index";
+@import "button/index";
```

Comment:
@CITguy, I think we should document the SCSS file structure rather than adding file placeholders.

Thoughts?